multiqc/modules/rsem/rsem.py

```diff
@@ -0,0 +1,161 @@
+            'title': '{} reads unalignable '.format(config.read_count_prefix),
+            'description': 'number of unalignable reads ({})'.format(config.read_count_desc),
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+
+        headers['Alignable'] = {
+            'title': '{} reads alignable'.format(config.read_count_prefix),
+            'description': 'number of alignable reads ({})'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+
+        headers['Filtered'] = {
+            'title': '{} reads filtered'.format(config.read_count_prefix),
+            'description': 'number of filtered reads due to too many alignments ({})'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+
+        headers['Unique'] = {
+            'title': '{} reads aligned uniquely '.format(config.read_count_prefix),
+            'description': 'number of reads aligned uniquely to a gene ({})'.format(config.read_count_desc),
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+        headers['Multi'] = {
+            'title': '{} reads aligned to multiple genes'.format(config.read_count_prefix),
+            'description': 'number of reads aligned to multiple genes ({})'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+        headers['Uncertain'] = {
+            'title': '{} reads aligned to multiple locations'.format(config.read_count_prefix),
+            'description': 'number of reads aligned  ({}) to multiple locations in the given reference sequences, which include isoform-level multi-mapping reads'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+        self.general_stats_addcols(self.rsem_mapped_data, headers)
+
+
+    def rsem_chart (self):
+        """ Make the rsem assignment rates plot """
+        
+        # Config for the plot
+        config = {
+            'id': 'rsem_assignment_plot',
+            'title': 'rsem Assignments',
+            'ylab': '# Reads',
+            'cpswitch_counts_label': 'Number of Reads'
+        }
+        log.debug(self.rsem_mapped_data)
+        log.debug(" " .join(self.rsem_keys))
+        return bargraph.plot(self.rsem_mapped_data, ['Unique','Multi','Uncertain'],config)
```

Comment:
Does `Unique`+`Multi`+`Uncertain` == `Alignable`? If so, could we add `Filtered` and `Filtered` to this plot so that the total count sums to the total number of reads?